components/Inputs/TileSelectInput.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { Text, TouchableOpacity } from 'react-native';
+import { FImage } from 'components/Composition/FImage';
+import placements from 'themes/placements';
+import sizes from 'themes/sizes';
+import colors from 'themes/colors';
+import fonts from 'themes/fonts';
+
+export const TileSelectInput = ({
+  iconSize, iconDefault, iconPressed, label, setValue, value,
+}) => {
+  const [
+    isPressed,
+    setIsPressed,
+  ] = useState(false);
+  return (
+    <TouchableOpacity onPress={() => setIsPressed(!isPressed)}>
+      <FImage
+        imagePath={isPressed ? iconPressed : iconDefault}
+        width={iconSize}
+        height={iconSize}
+        containerStyle={{
+          padding: sizes.PADDING_5,
+          margin: sizes.MARGIN_5,
+          backgroundColor: isPressed ? colors.PRIMARY : colors.LIGHT_GRAY,
+          borderRadius: sizes.RADIUS_10,
```

Comment:
jeżeli style nie sa uzależnione to przenieść je do const styles = StyleSheete.create({}) dlatego że tutaj niepotrzebnie zajmują miejsce i kod jest mniej czytelny